libkbfs/config_local.go

```diff
@@ -142,6 +144,47 @@ type ConfigLocal struct {
 	rekeyFSMLimiter *OngoingWorkLimiter
 }
 
+// DiskCacheMode represents the mode of initialization for the disk cache.
+type DiskCacheMode int
+
+var _ flag.Value = (*DiskCacheMode)(nil)
+
+const (
+	// DiskCacheModeOff indicates to leave off the disk cache.
+	DiskCacheModeOff DiskCacheMode = iota
+	// DiskCacheModeLocal indicates to use a local disk cache.
+	DiskCacheModeLocal
```

Comment:
I assume this is what `DiskBlockCacheStandard` is? Would it make sense to rename that type to `DiskBlockCacheLocal`?

Reply:
Yeah, I'll do that, thanks.